features/step_definitions/forum/topic_steps.rb

```diff
@@ -68,31 +70,31 @@
   step %(I press "Create thread")
 end
 
-When /^the (user "[^"]*") post in the topic in (the forum of "[^"]*")$/ do |user, forum|
+When "the {user} post in the topic in {forum}" do |user, forum|
   FactoryBot.create(:post, :topic => forum.topics.first, :forum => forum, :user => user)
 end
 
-Then /^(the forum of "[^"]*") should have topic "([^"]*)"$/ do |forum, title|
+Then "{forum} should have topic {string}" do |forum, title|
   assert_not_nil forum.topics.find_by_title(title)
 end
 
-Then /^(the forum of "[^"]*") should not have topic "([^"]*)"$/ do |forum, title|
+Then "{forum} should not have topic {string}" do |forum, title|
   assert_nil forum.topics.find_by_title(title)
 end
 
-Then /^(the forum of "[^"]*") should have topic "([^"]*)" in category "([^"]*)"$/ do |forum, title, category|
+Then "{forum} should have topic {string} in category {string}" do |forum, title, category|
   topic = forum.topics.find_by_title(title)
   assert_not_nil topic
   assert_equal category, topic.category.try!(:name)
 end
 
-Then /^(the forum of "[^"]*") should have sticky topic "([^"]*)"$/ do |forum, title|
+Then "{forum} should have sticky topic {string}" do |forum, title|
   topic = forum.topics.find_by_title(title)
   assert_not_nil topic
   assert topic.sticky?
 end
 
-Then /^(the forum of "[^"]*") should have non\-sticky topic "([^"]*)"$/ do |forum, title|
+Then "{forum} should have non-sticky topic {string}" do |forum, title|
```

Comment:
all step definitions with `{forum}`, to have the convention running, I think should be like
```
Then "the {forum} should have non-sticky topic {string}" do |forum, title|
```

See my previous comment about `{forum}`: https://github.com/3scale/porta/pull/2407/files#r695153651